dynamic_perks/mod_settings.nut

```diff
@@ -0,0 +1,2 @@
+local generalPage = ::DynamicPerks.Mod.ModSettings.addPage("General");
+generalPage.addColorPickerSetting("pergroup_highlight_color", "255,234,125,0.5", "Perk group highlighting", "Set the color of the perk group highlighting. To disable it, just set Alpha to zero.");
```

Comment:
Maybe also add a setting to disable this feature? I don't like the 'disable' being 0 alpha color

Reply:
I feel it's just an extra check for no real purpose. All this thing does is add a visual aid, opacity 0 removes the visual aid, 'disable' setting also removes the visual aid...

Reply:
Feels cleaner to me but ig that's fair.